src/services/api/fantlab/editions.ts

```diff
@@ -0,0 +1,19 @@
+import _ from 'lodash';
+
+export const url = '/search-ids';
+
+export const cache = true;
```

Comment:
Why do you think we need to cache that query?

Reply:
I don't think so. Just inaccurate copy-past